UIKit/Location Controller/TWTLocationController.m

```diff
@@ -0,0 +1,189 @@
+//
+//  TWTLocationController.m
+//  Toast
+//
+//  Created by Steve Foster on 2/3/15.
+//  Copyright (c) 2015 Two Toasters, LLC. All rights reserved.
+//
+
+#import "TWTLocationController.h"
+
+@import CoreLocation;
+
+static NSTimeInterval const kLocationTimeoutInterval = 60;
+NSString *const kLocationControllerErrorDomain = @"locationcontroller";
+
+
+@interface TWTLocationController () <CLLocationManagerDelegate>
+
+@property (nonatomic, strong) CLLocationManager *locationManager;
+@property (nonatomic, strong) CLLocation *location;
+@property (nonatomic, strong) NSMutableArray *requestBlocks;
```

Comment:
Make these readonly. `requestBlocks` can be created once and emptied each time you use it.